core/modules/filters/savers.js

```diff
@@ -0,0 +1,24 @@
+/*\
+title: $:/core/modules/filters/savers.js
+type: application/javascript
+module-type: filteroperator
+
+This Filter operator return savers title
+
+\*/
+;(function () {
```

Comment:
Please use the IIFE format used by other core modules (ie without the semicolon).

Reply:
General Comment below